bitcoind/client.go

```diff
@@ -519,6 +519,19 @@ func (c *Connector) SendTransaction(rawTx []byte) error {
 	return nil
 }
 
+// ConfirmedBalance returns number of funds available under control of
+// connector.
+//
+// NOTE: Part of the common.Connector interface.
+func (c *Connector) ConfirmedBalance(account string) (string, error) {
+	balance, err := c.client.GetBalanceMinConf(account, c.cfg.MinConfirmations)
+	if err != nil {
+		return "", err
+	}
+	return decimal.NewFromFloat(
```

Comment:
Shouldn't we return minimal denominator here, satoshis?

Reply:
Don't know, for now this is required format.